release/default.nix

```diff
@@ -333,17 +333,27 @@ jobs // {
 
     patchPhase = "touch .update-on-nixos-rebuild";
 
-    XZ_OPT = "-1";
     tarOpts = ''
-      --owner=0 --group=0 --mtime="1970-01-01 00:00:00 UTC" \
+      --owner=0 --group=0 \
+      --mtime="1970-01-01 00:00:00 UTC" \
       --exclude-vcs-ignores \
-      --transform='s!^\.!${name}!' \
     '';
 
     installPhase = ''
       mkdir -p $out/{tarballs,nix-support}
-      cd nixos
-      tar cJhf $out/tarballs/nixexprs.tar.xz ${tarOpts} .
+      tarball=$out/tarballs/nixexprs.tar
+
+      for file in nixos/* nixos/.*; do
+          if [ -f "$file" ]; then
+              tar uf "$tarball" --transform "s|^nixos|${name}|" ${tarOpts} "$file"
+          fi
+      done
```

Comment:
The glob pattern which matches on `nixos/.*` makes me uncomfortable. I don't know whether this will match the directory entry `nixos/..`, but this could lead to files outside this directory being accidentally included if it weren't for the `-f` test. I presume this works as intended, though if it were me I would probably implement this with find(1) instead.